src/app/image-group-page/image-group-page.component.ts

```diff
@@ -428,11 +438,41 @@ export class ImageGroupPage implements OnInit, OnDestroy {
       })
       .catch( error => {
         console.error(error)
-        this.exportLoadingStateopts.state = LoadingState.error
```

Comment:
@rehanabbasi This is all looking really solid.

My only concern is that it appears that we'll never be setting `this.exportLoadingStateopts.state = LoadingState.error` or setting `this.exportLoadingStateopts.errorType = 'server'` again, which is assuming that all exports will never actually fail even after polling the check endpoint.

This is used in the "try again" logic.